.circleci/config.yml

```diff
@@ -162,7 +162,7 @@ jobs:
       - install_js
       - run:
           name: '`yarn prettier` changes committed?'
-          command: yarn prettier check-changed
+          command: yarn prettier --check
```

Comment:
@m4theushw 
I've also brought [this change](https://github.com/mui/mui-x/pull/6876/commits/c66564f7efc6d8ddf8636e324360e8af34c8033a) in as to not introduce a pipeline regression after the monorepo/prettier change. 😉